src/app/admin/AdminConfig.ts

```diff
@@ -0,0 +1,21 @@
+export class AdminConfig {
+  // artist API
```

Comment:
we should have one base url in config file or env variable, 
and this file should not be in git ,
other option , we need somehow to distinguish between environment  ,